lib/chrome_remote/client.rb

```diff
@@ -12,31 +12,33 @@ def initialize(ws_url)
     def send_cmd(command, params = {})
       msg_id = generate_unique_id
 
-      ws.send_msg({method: command, params: params, id: msg_id}.to_json)
+      ws.send_msg({ method: command, params: params, id: msg_id }.to_json)
 
-      msg = read_until { |msg| msg["id"] == msg_id }
-      msg["result"]
+      msg = read_until { |msg| msg['id'] == msg_id }
+      msg['result']
     end
 
     def on(event_name, &block)
       handlers[event_name] << block
     end
 
-    def listen_until(&block)
-      read_until { block.call }
+    def listen_until
```

Comment:
Sorry about this - my editor automatically runs Rubocop on save and this must be an autofixed rule. If you feel strongly about keeping the old syntax, I can revert.